build.gradle

```diff
@@ -153,60 +113,17 @@ license {
 	exclude('**/*.json') // JSON files do not have headers
 }
 
-task checkAliases {
-	description 'Checks for the existence of the aliases.'
-
-	doLast {
-		def aliasFolder = new File('skript-aliases')
-		if (!aliasFolder.exists() || aliasFolder.listFiles().length == 0) {
-			throw new InvalidUserDataException("Aliases are missing from 'skript-aliases' folder. Consider fetching submodules with 'git submodule update --init'.")
-		}
-	}
-}
-
-jar.dependsOn checkAliases
-
-task sourceJar(type: Jar) {
-	from sourceSets.main.allJava
-}
-
-publishing {
-	publications {
-		mavenJava(MavenPublication) {
-			groupId "ch.njol"
-			artifactId "skript"
-			version project.property("version")
-
-			from components.java
-
-			artifact sourceJar {
-				classifier "sources"
-			}
-		}
-	}
-
-	repositories {
-		maven {
-			url "file:///itemapi_repo/"
-		}
-	}
-}
-
-// Set compile encoding to UTF-8 (#850)
-compileJava.options.encoding = 'UTF-8'
-compileTestJava.options.encoding = 'UTF-8'
-
 // Task to check that test scripts are named correctly
 tasks.create('testNaming') {
 	doLast {
 		// Regression tests
 		for (def file : project.file('src/test/skript/tests/regressions').listFiles()) {
 			def name = file.getName()
-			if (!name.toLowerCase().equals(name)) {
+			if (name.toLowerCase() != name) {
```

Comment:
why was this changed?

Reply:
in reality, i preferred to use this operator because i find it clearer. according to the [Groovy documentation](https://docs.groovy-lang.org/latest/html/documentation/core-operators.html#_identity_operator), it is also calling `equals`. but if you want me to revert this change, let me know!

Reply:
it's fine, I just forgot it calls equals in groovy